bindings/rust/wasmedge-sys/examples/async_host_func2.rs

```diff
@@ -0,0 +1,39 @@
+use wasmedge_sys::{Executor, FuncType, Function, Store};
```

Comment:
Please add the necessary description of how to build/run this example. If there was any assumption, please also make it clear to users. Thanks!